cypress/support/commands.js

```diff
@@ -0,0 +1,46 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+
+// Find the button to start the form
+Cypress.Commands.add("formStartButton", () => {
+    return cy.contains('Formulier starten');
+})
+
+// Find a form field by its key
+Cypress.Commands.add("formField", (fieldKey, timeout=10000) => {
+    return cy.get(`.openforms-form-control--${fieldKey}`, {timeout: timeout});
+})
+
+// Find the continue button
+Cypress.Commands.add("continueButton", (timeout=10000) => {
+    return cy.get('button[name="next"][type="submit"]', {timeout: timeout});
```

Comment:
I guess it would be better if these elements have data attributes to find them, that way we don't have to rely on these kind of CSS selectors

Reply:
I think I saw the NLX team use test IDs but I'm not sure